pkg/provider/provider.go

```diff
@@ -26,15 +26,15 @@ func Provider() *schema.Provider {
 			"username": {
 				Type:        schema.TypeString,
 				Optional:    true,
-				Description: "Materialize username. Can also come from the `MZ_USER` environment variable.",
-				DefaultFunc: schema.EnvDefaultFunc("MZ_USER", nil),
+				Description: "Materialize username. Can also come from the `MZ_USERNAME` environment variable.",
```

Comment:
Should this one be `user` and `MZ_USER`? PostgreSQL calls this parameter "user" rather than username, at least.

Reply:
Neat! +1 for `user` from me!
Tagging @dehume to see what his thoughts are as well

Reply:
`user` and `MZ_USER` sounds good